plugins/module_utils/network/eos/argspec/ospfv3/ospfv3.py

```diff
@@ -137,9 +137,28 @@ def __init__(self, **kwargs):
                                         "lsa": {"type": "bool"},
                                     },
                                 },
+                                "spf": {
+                                    "type": "dict",
+                                    "options": {
+                                        "max": {"type": "int"},
+                                        "initial": {"type": "int"},
+                                        "min": {"type": "int"},
+                                    },
+                                },
+                                "lsa": {
+                                    "type": "dict",
+                                    "options": {
+                                        "max": {"type": "int"},
+                                        "initial": {"type": "int"},
+                                        "min": {"type": "int"},
+                                        "direction": {
+                                            "type": "str",
+                                            "choices": ["rx", "tx"],
+                                        },
+                                    },
+                                },
                                 "out_delay": {"type": "int"},
                                 "pacing": {"type": "int"},
-                                "lsa": {"type": "int"},
```

Comment:
So this is the last backwards incompatible change, I think. I don't recall if you can quietly support taking an int instead of a dictionary (with a warning to say please change to `lsa: {"initial": <whatever>, "direction": "rx"}`).

If it's not possible to support both, I think you need to rename timers.lsa (and _maybe_ timers.spf to match?) and deprecate the old timers.lsa until it can be removed along with throttle.

Reply:
I have set type as "raw" for "lsa". A warning will be thrown if the value passed for "lsa" is not of type dict.